src/sinks/clickhouse.rs

```diff
@@ -190,9 +189,9 @@ impl RetryLogic for ClickhouseRetryLogic {
                 //
                 // Reference: https://github.com/timberio/vector/pull/693#issuecomment-517332654
                 if body.starts_with(b"Code: 117") || body.starts_with(b"Code: 53") {
-                    None
+                    RetryAction::DontRetry("incorrect data".into())
```

Comment:
Is this the clickhouse terminology? Ideally, a user should be able to google this and pretty quickly figure out what's going on.

Reply:
Yeah, we could attempt to print the code that we match on, that would be helpful. :+1: